edenai_apis/apis/amazon/tests/test_amazon_helpers.py

```
@@ -0,0 +1,273 @@
+            (50, 3.0),
+        ],
+    )
+    def test_volume_conversion(self, volume, expected):
+        assert amazon_speaking_volume_adapter(volume) == pytest.approx(expected)
+
+
+class TestGetRightAudioSupportAndSamplingRate:
+    @pytest.mark.unit
+    @pytest.mark.amazon
+    def test_default_format(self):
+        ext, fmt, rate = get_right_audio_support_and_sampling_rate("", 0)
```

Ruff (0.15.10)

[warning] Unpacked variable `rate` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)